.github/workflows/production-types.yml

```diff
@@ -88,5 +110,5 @@ jobs:
       - name: Open issue in nulib/repodev_planning_and_docs
         working-directory: "${{env.TMP_REPODEVDIR}}"
         run: |
-          gh issue create -t "Update nulib/dcapi-types for API release v${{ steps.package-version.outputs.current-version}}" -b "Review PR: ${{ steps.open-types-pr.outputs.pr}} and release nulib/dcapi-types if needed." -a mathewjordan -l "digital collections" -l "front end"
+          gh issue create -t "Update nulib/dcapi-types for API release v${{ steps.package-version.outputs.current-version}}" -b "Review PR: ${{ steps.open-types-pr.outputs.pr}} and release nulib/dcapi-types if needed." -l "digital collections" -l "front end"
```

Comment:
But the CODEOWNERS + teams won't affect the Github issue will it? (If not, let's not remove Mat otherwise the issue will fall into oblivion.)

Reply:
You're right, it doesn't affect the issue creation, but I wasn't sure how necessary/relevant that is if someone it reviewing the PR anyways. I'll add Mat back in for the issue